cost/google/idle_compute_instances/google_idle_compute_instances.pt

```diff
@@ -209,77 +216,102 @@ script "js_calculated_utilization", type: "javascript" do
   result "results"
   parameters "ds_compute_utilization", "ds_memory_utilization"
   code <<-EOS
-  results = []
-  for ( i =0; i < ds_compute_utilization.length; i++ ){
+  var results = []
+  for (i = 0; i < ds_compute_utilization.length; i++) {
     var instance_id = ds_compute_utilization[i].instance_id
-    if ( instance_id === null || instance_id === undefined ) {
-      // No instance id, continue
-    } else {
-      var points = ds_compute_utilization[i].cpu_points
-      if ( points === null || points === undefined ) {
-        var cpu_maximum = "101"
-        var cpu_average = "101"
-        var cpu_minimum = "101"
-      } else {
-        var cpu_maximum = parseFloat( points.reduce( function (x, y) { return Math.max(x, y) } )).toFixed(2)
-        var cpu_sum = _.reduce(points, function(memo, num){ return memo + num; }, 0);
-        var cpu_average = parseFloat(cpu_sum / points.length).toFixed(2)
-        var cpu_minimum = parseFloat( points.reduce( function (x, y) { return Math.min(x, y) } )).toFixed(2)
-      }
-      var memory_record = _.find(ds_memory_utilization, function(record) { return record.instance_id == instance_id; })
-      var memory_maximum = "101"
-        var memory_average = "101"
-        var memory_minimum = "101"
-      if ( memory_record === null || memory_record === undefined ) {
-        // No memory records, continue
-      } else {
-        if ( memory_record.mem_points === null || memory_record.mem_points === undefined ) {
-          // No memory records, continue
-        } else {
-          var memory_maximum = parseFloat( memory_record.mem_points.reduce( function (x, y) { return Math.max(x, y) } )).toFixed(2)
-          var memory_sum = _.reduce(memory_record.mem_points, function(memo, num){ return memo + num; }, 0);
-          var memory_average = parseFloat(memory_sum/memory_record.mem_points.length).toFixed(2)
-          var memory_minimum = parseFloat( memory_record.mem_points.reduce( function (x, y) { return Math.min(x, y) } )).toFixed(2)
-        }
-      }
-
-      results.push(
-        {
-          zone: ds_compute_utilization[i].zone,
-          projectId: ds_compute_utilization[i].projectId,
-          instance_id: instance_id,
-          cpu_average: cpu_average,
-          cpu_maximum: cpu_maximum,
-          cpu_minimum: cpu_minimum,
-          mem_average: memory_average,
-          mem_maximum: memory_maximum,
-          mem_minimum: memory_minimum
-        }
-      )
+    if (instance_id === null || instance_id === undefined) {
```

Comment:
Instead of using an if statement and continue (which won't work with my _.each recommendation I provided above), just do an if statement like this and include the rest of the code in the if block:

if (instance_id != null && instance_id != undefined) {